CHANGELOG.md

```diff
@@ -1,3 +1,301 @@
+## [Talos 1.9.0-alpha.0](https://github.com/siderolabs/talos/releases/tag/v1.9.0-alpha.0) (2024-10-18)
+
+Welcome to the v1.9.0-alpha.0 release of Talos!  
+*This is a pre-release of Talos*
+
+Starting with Talos v1.8.0, only standard assets would be published as github release assets. These include:
+
+* `cloud-images.json`
+* `talosctl` binaries
+* `kernel`
+* `initramfs`
+* `metal` iso and disk images
+* `talosctl-cni-bundle`
+
+All other release assets can be downloaded from [Image Factory](https://www.talos.dev/latest/talos-guides/install/boot-assets/#image-factory).
+
+Please try out the release binaries and report any issues at
+https://github.com/siderolabs/talos/issues.
+
+### `talosctl cgroups`
+
+The `talosctl cgroups` command has been added to the `talosctl` tool.
+This command allows you to view the cgroup resource consumption and limits for a machine, e.g.
+`talosctl cgroups --preset memory`.
+
+
+### Component Updates
+
+Linux: 6.6.57
+containerd: 2.0.0-rc.5
+Flannel: 0.25.7
```

Comment:
```suggestion
Linux: 6.6.57
containerd: 2.0.0-rc.5
Flannel: 0.25.7
systemd-udevd: 256.6 replaced eudev
```

Reply:
this should have been added in the PR replacing eudev, but I will add a note here ;)

Reply:
thanks!